apps/sms/views/conversation/style/message.css

```diff
@@ -142,6 +142,20 @@
   color: #d1eeee;
 }
 
+.message.incoming .message-details .late-arrival-notice {
+  color: #026a69;
+  font-size: 1.3rem;
+  display: flex;
+  align-items: center;
+}
+
+.message.incoming .message-details .late-arrival-notice:before {
+  content: url('images/icons/late_clock.png');
```

Comment:
@julienw I have managed to kept the pseudo element and use background-size at the same time. I tested it in mulet and it seems to work but you may double check this :)